.devcontainer/Dockerfile

```diff
@@ -11,4 +11,6 @@ RUN if [ "${NODE_VERSION}" != "none" ]; then su vscode -c "umask 0002 && . /usr/
 #     && apt-get -y install --no-install-recommends <your-package-list-here>
 
 # [Optional] Uncomment this line to install global node packages.
-# RUN su vscode -c "source /usr/local/share/nvm/nvm.sh && npm install -g <your-package-here>" 2>&1
\ No newline at end of file
+# RUN su vscode -c "source /usr/local/share/nvm/nvm.sh && npm install -g <your-package-here>" 2>&1
+
+RUN echo '\nsource /usr/share/bash-completion/completions/git' >> /home/vscode/.bashrc
```

Comment:
@sebbi08 helped me to add completions for git in my bash terminal in the documentation repository and adding this line in the Dockerfile was a possible solution. @jkoenig134 is there another solution or is it okay to add this specific line here?

Reply:
I don't care, but isn't this the wrong PR to add this?

Reply:
You got me. :D I thought it would be overkill if we opened a separate PR for this single line.  :)